test/vars/samples/implicit/_config.js

```diff
@@ -1,16 +1,5 @@
 export default {
 	test(assert, vars) {
-		assert.deepEqual(vars, [
-			{
-				export_name: 'foo',
-				injected: false,
-				module: false,
-				mutated: false,
-				name: 'foo',
-				reassigned: false,
-				referenced: true,
-				writable: true,
-			},
-		]);
+		assert.deepEqual(vars, []);
```

Comment:
Maybe delete this particular test altogether? It's kind of weird now.

Reply:
Or rename it maybe? if this sort of thing isn't covered in another test

Reply:
yep, renamed it